module_utils/kubevirt_common.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0 (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
@tripledes  Sergi, I'm not an expert in Licenses... k8s Ansible functions have "This is a part of Ansible.. " and a reference to GNU license, e.g. see [k8s/common.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/k8s/common.py) However, kubernetes itself has [Apache 2](https://github.com/kubernetes/kubernetes/blob/master/LICENSE). Question: should we put it as "This is a part of Ansible ..." too? 

@fabiand can you comment it please?